validator.py

```diff
@@ -113,6 +117,16 @@ def validate_one_level(self, level_name, to_validate, type_obj, required, api_ya
         is_valid = True
         found_keys = []
 
+        # do not require characters if persist_characters is true
+        if level_name == 'Scenes/State' and to_validate.get('persist_characters', False):
```

Comment:
This wasn't working for me because `to_validate` is the `State`, but `persist_characters` is defined at the `Scene` level.